libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -388,13 +390,23 @@ void AP_MotorsHeli_Single::move_actuators(float roll_out, float pitch_out, float
     }
 
     // ensure not below landed/landing collective
-    if (_heliflags.landing_collective && collective_out < _collective_land_min_pct && !_main_rotor.in_autorotation()) {
+    if (_heliflags.land_complete && _heliflags.landing_collective && collective_out < _collective_land_min_pct + _collective_offset_landed) {
+        _collective_offset_landed += 0.0001 * (_collective_land_offset_pct - _collective_offset_landed);
```

Comment:
Please add a comment to make it clear that this increases collective away from landed collective position.